--- configure.ac.orig	2017-02-04 11:31:11.949794798 -0500
+++ configure.ac	2017-02-04 11:31:27.726870712 -0500
@@ -136,7 +136,7 @@
 have_win32="no"
 lt_enable_auto_import=""
 case $host_os in
-mingw*)
+mingw*|*msys*)
         have_win32="yes"
         lt_enable_auto_import="-Wl,--enable-auto-import"
         ;;
@@ -208,7 +208,7 @@
 
 dnl Link with winsock for socket functions on MinGW
 case $host_os in
-        *mingw*)
+        *mingw*|*msys*)
                 AC_CHECK_LIB([ws2_32],[main])
                 ;;
         *)
